lib/coinbase/signer.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require_relative 'client'
+
+module Coinbase
+  # A representation of a Signer. Signers are assigned to sign transactions for a Wallet.
+  class Signer
+    # Returns a new Signer object. Do not use this method directly. Instead, use Signer.default.
+    def initialize(model)
+      @model = model
+    end
+
+    class << self
+      # Returns the default Signer for the CDP Project.
+      # @return [Coinbase::Signer] the default Signer
+      def default
+        response = Coinbase.call_api do
+          signers_api.list_server_signers
+        end
+
+        raise "No Signer's associated with the project" if response.data.empty?
+
+        new(response.data.first)
+      end
+
+      def signers_api
```

Comment:
The nicest part about exposing the class methods this way is you can make private ones:
```suggestion
      private

      def signers_api
```